### ISSUE TITLE
Unable to connect iOS Device

### ISSUE THREAD

**Issue by slewis30328** [airsdk/Adobe-Runtime-Support#3332]
Unable to see iOS device through IntelliJ or AIR SDK Utilities. 

We have a legacy mobile app and I have not made changes to it in a long time - at least a year. When I made last change, I had a different laptop (older version of Win 10) and had iTunes installed. Never had any issues connecting to device(s) with that setup. I now have "newer" laptop, updated iOS version, new devices, and don't have iTunes installed (don't think that's issue since docs say you no longer depend on those libraries). I have tried iPods, iPads, iPhones and no connectivity with AIR utilities. All devices popup in my Windows environment and can see files on device via Windows Explorer.

Command: air51\lib\aot\bin\iOSBin\ideviceinfo
Error Message: ERROR: No device found!

OS Windows: Windows 10 Enterprise - 19044.4529
AIR SDK: 51.0.1.3
iOS version: 17.5.1
Device: iPhone 15 Prod

Really appreciate any assistance!

Thanks

Steve

**Comment by ajwfrost** [comment]
Hi

We don't use the iTunes libraries now, instead we use a third party library/utility called "imobiledevice" (Windows builds are at http://docs.quamotion.mobi/docs/imobiledevice/). So that "ideviceinfo.exe" utility is from here..

And I think there is still some dependency on iTunes, at some low level, because I find that I can't detect my iPhone on Windows unless I've loaded iTunes and got it to spot the phone connection.

Sorry! Please try downloading iTunes from the Windows Store and then launch it, you can ignore any log-in requests (I think) and just wait to see if the phone icon appears... Sadly, iTunes appears to need to be running (or maybe, to have run at some point?) before the connection is detected, so it's not just a reliance on libraries that iTunes deploys, there seems to be some run-time dependency here too..

thanks

**Comment by slewis30328** [comment]
I can't connect to the Windows Store with the Windows 10 version I have.  It's a corp computer and Windows Store says it needs to be updated to connect. That's a major undertaking, with corp, to get that done.  

I was able to download iTunes through Apple.  After installing and launching, iTunes can't see iOS device. Went through iTunes diagnostics too, and no luck.  Says it can't see any USB ports and doesn't see any devices. Windows has no problem seeing them. Frustrating...to say the least.

I have a MAC mini and no problem connecting to iOS devices through the AIR utilities and via IntelliJ. I would prefer to work on this Windows machine though - long story.  Any other suggestions you might have...would appreciate. 

Thanks!

**Comment by FliplineStudios** [comment]
Any luck with using `adt -devices -platform iOS` to see if your device shows up there?  My iPhone 15 shows up using that with AIR 51 on Windows for me (also works in 33.1, 50.2, etc.), and can use `adt -installApp` using the device handle shown.

**Comment by slewis30328** [comment]
I have tried that as well and get: **No connected device found.**

Are you aware of any security software that might cause this? Since I can see in Windows, that rabbit hole doesn't seem worth going down.

**Comment by slewis30328** [comment]
I have not been able to find the windows build for imobiledevice. Can you re-check your URL?  Still having issues.  Are you aware of any iTunes logs that might be lead to what the issue is? Thanks.

**Comment by ajwfrost** [comment]
Hmm.... it seems to have disappeared. And when we go to the original site:
https://libimobiledevice.org/
and scroll down to "Getting Started", the Windows support is missing.

It may be feasible to try building the source code for Windows.. but if iTunes can't see the phone, I don't hold out much hope of third party software detecting it. I would have suggested checking iTunes (is a corporate machine? might it be blocking USB access like this?) -> is the device set up for USB-based debugging/development? etc.

thanks

**Comment by slewis30328** [comment]
Yeah - looked all over for the windows build and couldn't find anything. Yes, it's a corp machine. The frustrating thing is I can see the iPhone with no problem through Windows. Message pops up if I want to allow device to access photos, etc. Say Yes to that and have to enter my iPhone password - all good.  Can see all files on phone.  However, iTunes nothing.  See's no devices. I've been through all the Apple tech notes and nothing works. Same result when I try running the AIR utilities. I've asked corp IT security about it and they said nothing should be blocking it. I do have Windows 10 - and I'm unable to access Windows Store without an update - says the error message.  Not sure if it's tied to that or not.  I guess it's time to grab a Windows 11 machine and see if that will work. However, that doesn't make a lot of sense though - have another dev I work with at comp and he has Windows 10 - same corp build - and he can access his phone from iTunes. So...I don't know.

**Comment by amorganiv** [comment]
Extracting/Installing the Apple Mobile Device Support MSI 64 from iTunes did the trick on Windows 11.

How to install Apple Mobile Device Support without iTunes on Windows 10:
https://www.youtube.com/watch?v=g-Pw-lmIHG4
